k8s-test-runner/runner/runner.go

```diff
@@ -2,6 +2,7 @@ package runner
 
 import (
 	"context"
+	// "embed"
```

Comment:
Should we remove it?

Reply:
@skudasov otherwise it complains about `_ "embed"`